TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateChallengePayload.swift

```diff
@@ -35,15 +35,19 @@ public struct UpdatePushChallengePayload: UpdateChallengePayload {
   public let challengeSid: String
   //New status of the Challenge
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/c51c4513007d818dab4b64cd25c837959c7ec03d/TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateChallengePayload.swift#L36"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>UpdateChallengePayload.swift:36</code></td>
    </tr>
  </tbody>
</table>